interfaces/builtin/audio_playback.go

```diff
@@ -106,7 +106,7 @@ owner /{,var/}run/pulse/** rwk,
 # Shared memory based communication with clients
 /{run,dev}/shm/pulse-shm-* mrwk,
 
-owner /run/pulse/native/ rwk,
+owner /run/pulse/native/ rk,
```

Comment:
This rule looks to double up with a previous rule here:

https://github.com/snapcore/snapd/blob/92cf5ea5b3699271c3e5444a2b2e1dd9dba0862c/interfaces/builtin/audio_playback.go#L104

It also looks kind of weird, since I wouldn't expect `/run/pulse/native` to be a directory. As I understand it this is the equivalent of `$XDG_RUNTIME_DIR/pulse/native` when pulseaudio is run in system-wide mode, so it'd be a socket. In which case the rule wouldn't match due to the trailing slash. So I wonder if it can simply be removed?

Whatever you do, it would be good to make the same change to the legacy pulseaudio interface.

Reply:
Mmm... you are right, it's quite odd. How can I test this? I've been checking how to use pipewire in system-wide mode, and it's quite convoluted...

Reply:
I tested it with firefox and telegram, both in a classic ubuntu and core desktop, and works fine. But, of course, I haven't tested with system-wide pipewire daemon...